### PR TITLE
Use more meaning full popup header messages

### PR DESCRIPTION
**Pull request by ehsan-fj**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -77,38 +78,6 @@
               app.membershipCredential = {};
               clearOutput();
             },
-
-            discloseEmailAndName: function () {
```
Comment (rafaelbernard) [inline src/index.html:81]: Is that dead code?

File: src/irma-functions.js [hunk 2]
```diff
@@ -45,7 +52,7 @@ function irmaDoSession (request) {
         // Front-end options
         language: 'en',
         translations: {
-            header: 'Sign the agreement with <i class="irma-web-logo">IRMA</i>',
+            header: `${header} <i class="irma-web-logo">IRMA</i>`,
```
Comment (rafaelbernard) [inline src/irma-functions.js:55]: Hum... Maybe we can add a little InternetNZ icon before the message later... Would be cool.
`<i class="inz-logo">INZ</i> ${header} <i class="irma-web-logo">IRMA</i>`

**Review by rafaelbernard — APPROVED** [review]

File: src/index.html (hunk 1 above)
Comment (ehsan-fj) [reply src/index.html:81]: it's not used in that page! we just issue in that page not disclose! Am I right?

File: src/irma-functions.js (hunk 2 above)
Comment (ehsan-fj) [reply src/irma-functions.js:55]: Will be awesome :)

File: src/index.html (hunk 1 above)
Comment (rafaelbernard) [reply src/index.html:81]: you are right!

**Pull request merged**